crates/e2e/tests/e2e/liquidity.rs

```diff
@@ -200,6 +207,9 @@ fn create_zeroex_liquidity_orders(
         // fully covers execution costs
         maker_amount: order_creation.buy_amount.as_u128() * 3,
         taker_amount: order_creation.sell_amount.as_u128() * 2,
+        // makes 0x order partially fillable, but the amount is higher than the cowswap order to
```

Comment:
Every 0x order is partially fillable.

```suggestion
        // makes 0x order partially filled, but the amount is higher than the cowswap order to
```